src/ConfigCatClientOptions.ts

```diff
@@ -56,7 +76,11 @@ export abstract class OptionsBase implements IOptions {
     }
 
     getUrl(): string {
-        return this.baseUrl + "/configuration-files/" + this.apiKey + "/config_v4.json";
+        return this.baseUrl + "/configuration-files/" + this.apiKey + "/config_v5.json";
+    }
+
+    getCacheKey(): string {
+        return "config_v5_" + this.apiKey;
```

Comment:
Because you refactor the cache handling I'm just thinking can it help if we add library version to key...if some version sticks in the cache the new version can fix it...what do you think?

Reply:
I'm not sure if a new version should mean a "cache clean". I think it is enough If the config file version is different. If we want an explicit cache clean, we can always set here a new cache key so the new version will start from an empty cache.